packages/react-server/src/ReactFlightServer.js

```diff
@@ -4109,7 +4289,7 @@ function renderDebugModel(
     }
 
     if (counter.objectLimit <= 0 && !doNotLimit.has(value)) {
-      // We've reached our max number of objects to serialize across the wire so we serialize this
+      // We've reached our max number of objects to serializeDebug across the wire so we serializeDebug this
```

Comment:
Search & replace mistake?
```suggestion
      // We've reached our max number of objects to serialize across the wire so we serialize this
```